eth/taiko_api_backend.go

```diff
@@ -74,24 +74,42 @@ func NewTaikoAuthAPIBackend(eth *Ethereum) *TaikoAuthAPIBackend {
 	return &TaikoAuthAPIBackend{eth}
 }
 
-// BuildTxList initiates the process of building tx lists.
-func (a *TaikoAuthAPIBackend) BuildTxList(
-	beneficiary common.Address,
+// CHANGE(limechain):
+
+// UpdateConfigAndSlots updates slots and config params at the beginning of a new slot.
```

Comment:
It's not necessary this to be called at the beginning of a new slot, it can be called even earlier, when you know the epoch. Timestamp of the slot is deterministic